doc/quirks.md

```diff
@@ -8,3 +8,4 @@
 - The client wants to receive the last snapshot part last, otherwise the
   resulting delta is too long.
 - CCharacterCore has unused fields `m_HookDx`, `m_HookDy`
+- The packet payload can be empty. This happens when the flush happens and the resend flag is set but there were no chunks queued yet.
```

Comment:
"The flush" and "the resend flag" outside of the packet header are implementation details of Teeworlds. Perhaps you could phrase it like "This happens when chunks from the peer are lost, a resend of these is requested but no new chunks to the peer are queued yet."